src/plugins/intel_cpu/src/nodes/kernels/stack_allocator.hpp

```diff
@@ -0,0 +1,474 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <ie/ie_common.h>
+#include <cpu/x64/jit_generator.hpp>
+
+namespace ov {
+namespace intel_cpu {
+
+class StackAllocator final {
+public:
+    using jit_generator = dnnl::impl::cpu::x64::jit_generator;
```

Comment:
Such change make it inconsistent with `namespace x64 = dnnl::impl::cpu::x64;` in RegisterPool file

Reply:
Actually that would be great to get read of this alias in RegisterPool as well, just to avoid possible namespace conflicts, but in case of RegisterPool it is so tightly coupled with cpu_isa_t, that we should leave it for a while ad try to address this issue on the `jit_base` level.